tests/components/deconz/test_switch.py

```diff
@@ -140,7 +130,37 @@ async def test_switches(hass):
         await hass.async_block_till_done()
         set_callback.assert_called_with("put", "/lights/1/state", json={"on": False})
 
-    warning_device_device = gateway.api.lights["3"]
+    await gateway.async_reset()
+
+    assert len(hass.states.async_all()) == 0
+
+
+async def test_sirens(hass):
+    """Test that siren entities are created."""
+    data = deepcopy(DECONZ_WEB_REQUEST)
+    data["lights"] = deepcopy(SIRENS)
+    gateway = await setup_deconz_integration(hass, get_state_response=data)
+
+    assert len(hass.states.async_all()) == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)